README.md

```diff
@@ -1,10 +1,39 @@
 [![BCH compliance](https://bettercodehub.com/edge/badge/legalthings/lto-network-frontend?branch=master)](https://bettercodehub.com/)
 
-# LtoNetworkFrontend
+# LTO Network Frontend
+<a href="https://github.com/legalthings/lto-network-frontend/"><img src="https://lto.network/img/meta.png"></a>
+<!-- This project was generated with [Angular CLI](https://github.com/angular/angular-cli) using [Nrwl Nx](https://nrwl.io/nx).  -->
 
-This project was generated with [Angular CLI](https://github.com/angular/angular-cli) using [Nrwl Nx](https://nrwl.io/nx).
+## The Goal of Angular CLI
 
-## Nrwl Extensions for Angular (Nx)
+The project is a momorepo based project. 
+By now there are two main projects:
+1. Explorer
+1. Wallet
+
+## Requirements
+
+Before cloning the project you would need:
+1. `git` isntalled locally in your computer
+1. `npm` installed locally in your computer. It can be done with this `npm install npm@latest -g`. [Check npm documentation](https://docs.npmjs.com/)
+1. Make sure that you have Node 10.9 or later installed. See instructions [here](https://nodejs.org/en/download/). The Angular CLI requires Node 8, but development requires Node 10.
+1. Install Angular CLI. It can be done with `npm install -g @angular/cli`. The Angular CLI is a command-line interface tool that you use to initialize, develop, scaffold, and maintain Angular applications. You can use the tool directly in a command shell, or indirectly through an interactive UI such as Angular Console. [Angular CLI reference](https://angular.io/cli#cli-command-reference)
+
+## Installation
+
+To get started locally, follow these instructions:
+If you haven't done it already, make a fork of this [repo](https://github.com/legalthings/lto-network-frontend/fork).
+1. Clone to your local computer using `git`.
+1. Make sure that you have Node 10.9 or later installed. See instructions [here](https://nodejs.org/en/download/). The Angular CLI requires Node 8, but development requires Node 10.
+1. Enter project main directory and install npm modules (dependecies): `npm install`.
+
+## Getting Started - Local Development
+
+1. If you want to run `explorer` project, you can just use `ng serve`.
```

Comment:
To start explorer you need to use `ng serve explorer`. It just happened to be that explorer is a default project.